content/pages/about.html

```diff
@@ -0,0 +1,396 @@
+                </span>
+              </a>
+
+              {%- set header_2 = "Build automatically" %}
+              {%- set icon_2 = "fa-gears" -%}
+              <a href="#building" class="ui small header item">
+                <i class="fad {{ icon_2 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_2 }}
+                  <span class="sub header">Update your documentation whenever you push changes</span>
+                </span>
+              </a>
+
+              {%- set header_3 = "Preview your changes" %}
+              {%- set icon_3 = "fa-code-compare" -%}
+              <a href="#review" class="ui small header item">
+                <i class="fad {{ icon_3 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_3 }}
+                  <span class="sub header">Verify changes to your documentation during review</span>
+                </span>
+              </a>
+
+              {%- set header_4 = "Everything in one place" %}
+              {%- set icon_4 = "fa-box-check" -%}
+              <a href="#hosting" class="ui small header item">
+                <i class="fad {{ icon_4 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_4 }}
+                  <span class="sub header">We manage building, hosting, authentication, search, and analytics for you</span>
+                </span>
+              </a>
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </div>
+  </div>
+</section>
+
+{% macro about_divide(text=None) %}
+  <div class="ui text container">
+    <div class="ui {%- if text %} horizontal {%- endif %} divider">
+      {% if text %}{{ text }}{% endif %}
+    </div>
+  </div>
+{% endmacro %}
+
+{{ about_divide() }}
+
+<section>
+  <div class="ui very padded container">
+    <div class="ui vertically padded centered grid" id="authoring">
+
+      {% call about.step(
+          header=header_1,
+          image="/images/about/authoring.png",
+          image_alt="Terminal window showing commands to commit documentation changes",
+          icon=icon_1) %}
+        {% markdown %}
+        Manage software documentation using all of the **same tools and workflow**
+        you're already using for development. Write docs with your favorite editor,
+        commit changes using version control, and preview changes during review.
+        {% endmarkdown %}
+      {% endcall %}
+
+      {% call homepage.feature_detail_item(header="Versioning", icon="fad fa-layer-group") %}
+        Give your users documentation that won't go out of date.
+        Build multiple versions from commits, branches, or tags in your repository.
+      {% endcall %}
+
+      {% call homepage.feature_detail_item(header="Localization", icon="fad fa-language") %}
+        Integrate with localization tools to translate your documentation in multiple languages.
+      {% endcall %}
```

Comment:
I don't think this is a top-level feature to me. Almost none of our commercial users use this. I will have to think more about what to highlight here, but I would put custom domains, CDN, or search before localization

Reply:
Aye, I didn't feel strongly about this, but localization is used in community though, and this page is pulling double duty for both platforms.

If we have another authoring feature or highlight, agreed this doesn't need to be mentioned at this level. But I included it here as it is an authoring feature, and compliments multiple version support. Custom domains, cdn, search are all hosting features mentioned in the hosting section or below.